### PR TITLE
IPROTO-158 Remove autoImportClasses

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/IPROTO-158

### CODE REVIEW

File: processor/src/main/java/org/infinispan/protostream/annotations/impl/processor/ProtoSchemaAnnotation.java [hunk 1]
```diff
@@ -20,15 +19,14 @@ class ProtoSchemaAnnotation {
    private final String schemaPackageName;
    private final boolean service;
    private final ProtoSyntax syntax;
-   private boolean allowNullFields;
+   private final boolean allowNullFields;
```
Comment (rigazilla) [inline processor/src/main/java/org/infinispan/protostream/annotations/impl/processor/ProtoSchemaAnnotation.java:22]: seems unrelated. is it ok?

Comment (tristantarrant) [reply processor/src/main/java/org/infinispan/protostream/annotations/impl/processor/ProtoSchemaAnnotation.java:22]: Yeah, but it's such a tiny thing that I didn't want to create a different PR for that

**Comment by rigazilla** [comment]
merged thanks!
